app/src/main/res/layout/fragment_add_login.xml

```diff
@@ -111,7 +112,7 @@
         android:colorControlActivated="?attr/textPrimary"
         android:textColor="?attr/textPrimary"
         android:contentDescription="@string/saved_login_username_description"
-        app:layout_constraintEnd_toEndOf="@id/usernameHeader"
+        app:layout_constraintEnd_toStartOf="@id/usernameHeader"
```

Comment:
I don't think this should have changed? This is still displaying ok is because of the set width which should probably use 0dp as the constraints look set to anchor with usernameHeader